install-parser.sh

```diff
@@ -1,5 +1,5 @@
 set -eu
-PARSER="stanford-corenlp-full-2015-12-09"
+PARSER="stanford-corenlp-full-2016-10-31"
```

Comment:
Are some of the new options only available in the new parser? If so, we should try to catch expected errors from using an old version?